docs/create_and_deploy_jh_image.md

```diff
@@ -0,0 +1,160 @@
+# How to create a Jupyterhub image and deploy it on ODH
+
+This howto is about getting your repository built into an image and having that image made available in (your) Jupyterhub instance.
+
+We are going to deal with 2 issues:
+
+ 1. Have an image build and added to a public image repository.
```

Comment:
Can you modify this to - "image built" if build is used as a verb or 
" Have an image build and get it added ..." if image build is a noun here